l10n.toml

```diff
@@ -2,8 +2,15 @@
 basepath = "."
 
 locales = [
-
-  ]
+    "de",
+    "es",
+    "fr",
+    "it",
+    "ja",
+    "ko",
+    "zh-CN",
+    "zh-TW",
+]
```

Comment:
Is this how we will control what kind of locales get imported?

Reply:
This is what c-l uses by default as the locales it tests.

The concrete flow of adding a new locale or removing one is something we need to drill into still. In particular around the role of the pontoon admin UI, and who's driving this. And if there's more than one way to do it.